x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -7,18 +7,36 @@
 
 import { schema, TypeOf } from '@kbn/config-schema';
 
+const BaseActionRequestSchema = {
+  /** A list of endpoint IDs whose hosts will be isolated (Fleet Agent IDs will be retrieved for these) */
+  endpoint_ids: schema.arrayOf(schema.string(), { minSize: 1 }),
```

Comment:
I really want to make these camelCase but kept it since we need to support existing isolate/release